Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1409,7 +1414,7 @@ - (void)handleInitSuccess {
                 [self.deepLinkPresentingController presentViewController:branchSharingController animated:YES completion:NULL];
             }
         }
-    }
+    }    
```

Comment:
nitpicking and definitely not important, but if you do more cleanup you can pull out the extra whitespace here